__tests__/app.test.js

```diff
@@ -317,4 +317,23 @@ describe('GET /api/articles (topic query)', () => {
             expect(body.msg).toBe('Invalid topic!');
         })
     })
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('FEATURE REQUEST: responds with an article object with a new column called comment_count', () => {
+        const testArticle = {
+            author: expect.any(String),
+            title: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String)
+        }
```

Comment:
Asa it is a single article, would be nice to explicitly test the values. At the very least, the article id and comment count values

Reply:
Okaaaay done!!